ir-sanguinem-et-lacrimis/common/technologies/artillery.txt

```diff
@@ -1817,6 +1817,8 @@ technologies = {
 
 		research_cost = 1
 
+		dependencies = { antitank3 = 1 }
```

Comment:
has nothing to do with the speed issue; will include this in the bureau_and_tactics pull